fineract-provider/src/main/java/org/apache/fineract/interoperation/data/InteropTransactionData.java

```diff
@@ -118,6 +118,7 @@ public static InteropTransactionData build(SavingsAccountTransaction transaction
 
         StringBuilder sb = new StringBuilder();
         int currLength = 0;
+        /* 
```

Comment:
This looks a breaking change to me

Reply:
Done! the code has been updated to the service `InteropServiceImpl`